kinto/core/testing.py

```diff
@@ -1,34 +1,29 @@
-from collections import defaultdict
-import mock
 import os
-import threading
-import functools
+from collections import defaultdict
 
 try:
     import unittest2 as unittest
-except ImportError:
+except ImportError:  # pragma: no cover
```

Comment:
unittest2 is a backport of the new features added to the unittest testing framework in Python 2.7 

https://pypi.python.org/pypi/unittest2